### PR TITLE
(docs): README

### PR DESCRIPTION
**Pull request by lbenie**
+ adds review ninja badge

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/bawt/animated/pull/5" target="_blank">ReviewNinja

File: README.md [hunk 1]
```diff
@@ -4,5 +4,5 @@
 [![devDependency Status](https://david-dm.org/bawt/animated/dev-status.svg)](https://david-dm.org/bawt/animated#info=devDependencies)
 [![Issue Stats](http://issuestats.com/github/bawt/animated/badge/issue)](http://issuestats.com/github/bawt/animated)
 [![Issue Stats](http://issuestats.com/github/bawt/animated/badge/pr)](http://issuestats.com/github/bawt/animated)
```
Comment (lbenie) [inline README.md]: Nice addition to the README

File: README.md [hunk 2]
```diff
@@ -4,5 +4,5 @@
...
-
+[![ReviewNinja](https://app.review.ninja/54779798/badge)](https://app.review.ninja/bawt/animated)
```
Comment (lbenie) [inline README.md]: !completed Badge added :+1:

File: README.md (hunk 1 above)
Comment (lbenie) [inline README.md]: !fix

Comment (lbenie) [inline README.md]: !fixed

**Pull request merged**